### ISSUE TITLE
NG网页间歇性卡住几秒

### ISSUE THREAD

**Issue by Travel2Here** [mayswind/AriaNg#676]
下载中页面实时状态会不规律的间隔几秒卡顿 是什么原因？难道是写入的时候影响进度报告的传输？有没有人遇到？咋解决？

**Comment by nuitlejour** [comment]
存在卡顿现象, 尤其是下载数量20+的时候.

**Comment by lee850220** [comment]
經過實驗
本以為是連線數過多的問題，每個任務從128連線調降為32 => 仍會發生
任務數過多問題，限制單一任務 => 仍會發生
限制傳輸速度 => 不發生

最後觀察到只要流量超過30MB/s，就有機率卡頓，隨著速度上升會愈發嚴重，60MB/s的時候UI會卡死直到速度降下來

**Comment by Travel2Here** [comment]
> 經過實驗
> 
> 本以為是連線數過多的問題，每個任務從128連線調降為32 => 仍會發生
> 
> 任務數過多問題，限制單一任務 => 仍會發生
> 
> 限制傳輸速度 => 不發生
> 
> 
> 
> 最後觀察到只要流量超過30MB/s，就有機率卡頓，隨著速度上升會愈發嚴重，60MB/s的時候UI會卡死直到速度降下來，甚至會顯示斷線，但後台仍然可以正常使用指令
> 
> 

没错 我的个人观察也是超30M就非常卡了 看来确实是这个工具前端页的问题而非CPU或者硬盘是瓶颈 希望有能力改善的朋友出手 😁
